docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM python:3.7
+
+# Better movement with [Ctrl + →] and [Ctrl + ←]
+ENV INPUTRC=/etc/inputrc
+
+RUN set -ex \
+  && echo '"\e[1;5C": forward-word' >> /etc/inputrc \
+  && echo '"\e[1;5D": backward-word' >> /etc/inputrc
+
+# Set working directory.
+# Mount your project under the same path.
+WORKDIR /usr/src
+
+# Install useful programs
+RUN apt-get update -y && apt-get install -y \
+      less \
+      git
```

Comment:
What's up with this odd formatting?

Reply:
Readability, you can see at first glance what packages are being installed.